pkgs/memory_usage/lib/src/_util.dart

```diff
@@ -0,0 +1,7 @@
+// Copyright 2023 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+extension SizeConversion on int {
+  int mbToBytes() => this * 1024 * 1024;
```

Comment:
a nit, since this is existing code, but this should likely be a getter.

Reply:
I prefer not to change copied code as it is proven to work and it is not test covered in leak_tracker that much.